bench/c3c/optimized/types.c.ll

```diff
@@ -4121,7 +4121,7 @@ define dso_local void @type_setup(ptr nocapture noundef readonly %0) local_unnam
   %.sroa.317.0.extract.trunc.i.i = trunc nuw i64 %.sroa.317.0.extract.shift.i.i to i32
   %5 = shl i32 %.sroa.015.0.extract.trunc.i.i, 13
   %6 = and i32 %5, 16711680
-  %.not.i.i = icmp eq i32 %.sroa.317.0.extract.trunc.i.i, 0
+  %.not.i.i = icmp ult i64 %4, 4294967296
```

Comment:
Regression.

Reply:
Duplicate of https://github.com/dtcxzyw/llvm-opt-benchmark/pull/600#discussion_r1597405950